src/test/java/seedu/duke/person/PersonLogTest.java

```diff
@@ -0,0 +1,64 @@
+package seedu.duke.person;
+
+import org.junit.jupiter.api.Test;
+import seedu.duke.exceptions.StorageOperationException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.duke.testutil.SamplePersons.ALICE;
+import static seedu.duke.testutil.SamplePersons.BOB;
+import static seedu.duke.testutil.SamplePersons.JACK_NO_PHONE;
+import static seedu.duke.testutil.SamplePersons.JOHN;
+import static seedu.duke.testutil.SamplePersons.JOHN_DIFF_ID;
+import static seedu.duke.testutil.SamplePersons.MARY;
+
+class PersonLogTest {
+    PersonLog personLog = PersonLog.getInstance();
+
+    @Test
+    public void personLogTest() {
+        try {
+            personLog.clearAllPersons();
+            personLog.addPerson(ALICE);
+            personLog.addPerson(BOB);
+            personLog.addPerson(JOHN);
+            personLog.addPerson(MARY);
+            personLog.addPerson(JACK_NO_PHONE);
+            personLog.saveAllPersons();
+            personLog.loadAllPersons();
+        } catch (StorageOperationException e) {
```

Comment:
Is it possible to separate the save all persons and load all persons test to 2 separate tests?

Reply:
I tried to do it but had synchronisation errors, I will try again later on!